test/client/src/lib.rs

```diff
@@ -128,20 +136,59 @@ impl DefaultTestClientBuilderExt for TestClientBuilder {
 }
 
 fn genesis_config(changes_trie_config: Option<ChangesTrieConfiguration>) -> GenesisConfig {
-	GenesisConfig::new(
-		changes_trie_config,
-		vec![
-			sr25519::Public::from(Sr25519Keyring::Alice).into(),
-			sr25519::Public::from(Sr25519Keyring::Bob).into(),
-			sr25519::Public::from(Sr25519Keyring::Charlie).into(),
-		],
-		vec![
-			AccountKeyring::Alice.into(),
-			AccountKeyring::Bob.into(),
-			AccountKeyring::Charlie.into(),
-		],
-		1000,
-		Default::default(),
-		Default::default(),
+	cumulus_test_service::local_testnet_genesis(changes_trie_config)
+}
+
+fn additional_storage_with_genesis(genesis_block: &Block) -> BTreeMap<Vec<u8>, Vec<u8>> {
```

Comment:
I don't think that we need this entire function?

Reply:
Yes we do in line ~95